packages/editor/src/components/tabs/task/error/ErrorSelect.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import ErrorSelect from './ErrorSelect';
+import { ClientContext, ClientContextInstance, DataContext, DataContextInstance } from '../../../../context';
+import userEvent from '@testing-library/user-event';
+
+describe('ErrorSelect', () => {
+  function renderSelect(options?: { error?: string }) {
```

Comment:
maybe the test and the logic for it could look less duplicated if we hide/show a common wrapper for all these widgets in expiry?